lib/internal/errors.js

```diff
@@ -821,8 +821,11 @@ function hideInternalStackFrames(error) {
 // to make usage of the error in userland and readable-stream easier.
 // It is a regular error with `.code` and `.name`.
 class AbortError extends Error {
-  constructor() {
-    super('The operation was aborted');
+  constructor(message = 'The operation was aborted', options = undefined) {
+    if (options !== undefined && typeof options !== 'object') {
```

Comment:
`validateObject`?

Reply:
https://github.com/nodejs/node/pull/41008#discussion_r758007329

Reply:
We can't use validateObject here. validators.js imports errors.js which would set up a circular dependency and it's not worth the additional complexity of a lazy load.